cmd/account/mgmt/account-assign.go

```diff
@@ -97,10 +95,10 @@ func (o *accountAssignOptions) run() error {
 		rootID          string
 	)
 
-	if o.payerAccount == "osd-staging-1" {
+	if o.payerAccount == "osd-staging-1" || os.Getenv("AWS_ACCOUNT_NAME") == "osd-staging-1" {
```

Comment:
Can we pull `osd-staging-1` and the osd-staging-2 string out into constants?

Only adding the comment once but this would apply to all of the files contained within this PR.

Reply:
Piggy backing here - can we also pull "AWS_ACCOUNT_NAME" out as a constant?